bundler/lib/bundler/templates/newgem/lib/newgem.rb.tt

```diff
@@ -2,7 +2,14 @@
 
 require_relative "<%= File.basename(config[:namespaced_path]) %>/version"
 <%- if config[:ext] -%>
-require_relative "<%= File.basename(config[:namespaced_path]) %>/<%= config[:underscored_name] %>"
+# Attempt to load a versioned extension based on the Ruby version.
```

Comment:
Can you remove these changes from template? That workaround is only for fat gem, not generally usage.